.github/workflows/swaggerhub-publish.yml

```diff
@@ -30,6 +30,43 @@ jobs:
         with:
           name: specfile
           path: openapi/build/openapi.yml
+
+  testspec:
+    name: Test generated spec against the Moira API
+    runs-on: ubuntu-latest
+    needs: mergespec
+    strategy:
+      matrix:
+        python-version: [3.8]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: moira-alert/moira
```

Comment:
I think it's better to use this repo instead `https://github.com/moira-alert/docker-compose/blob/master/docker-compose.yml`

Reply:
Don't know if it makes sense, but I was thinking we need to test against the same branch name on main Moira API, for example:
`feature/add-teams` branch on `doc` repo should test on `feature/add-teams` on `moira` repository. 
Because if an API endpoint doesn't exist on master but is documented, then the test will fail (with 404 error)

Reply:
> `feature/add-teams` branch on `doc` repo should test on `feature/add-teams` on `moira` repository.

On second thoughts, this brings up the case when the `feature` is for `doc` only and doesn't concern the main Moira repo. I think I'd just go ahead and change it to the one you suggested.